### ISSUE TITLE
Problems finding `fuzzyc2cpg.sh` on some platforms

### ISSUE THREAD

**Issue by astrogeco** [ShiftLeftSecurity/joern#466]
Followed installation instructions in docs. Then running sample project

```
joern> importCode(inputPath="./x42/c", projectName="x42-c")  
Using generator for language: C
Creating project `x42-c` for code at `./x42/c`
Support for this language is only available in ShiftLeft Ocular with an appropriate license
res0: Option[Cpg] = None
```

Also tried zip-file workaround in #387 using v1.1.112 without success.
See https://github.com/ShiftLeftSecurity/joern/issues/387#issuecomment-731735464

**Comment by dongshen** [comment]
i got the same issue

**Comment by karanpanjabi** [comment]
Are you sure you started joern by typing ./joern in the joern-cli directory you downloaded? 
Other way to get around this is to use joern-parse on the source code directory, following by importCpg in the joern shell

**Comment by fabsx00** [comment]
Looking into it. Just to clarify, this isn't about having an appropriate licence, what it means is that joern can't find the shell script `fuzzyc2cpg.sh` anywhere. If you could tell us exactly how you installed joern and on what platform, we have a better chance of reproducing this.

**Comment by fabsx00** [comment]
Since this problem keeps on popping up, I'll rewrite this component to use the Java classes directly instead of calling the CPG generator via the shell.

**Comment by fabsx00** [comment]
https://github.com/ShiftLeftSecurity/codepropertygraph/pull/1128

**Comment by fabsx00** [comment]
Should be fixed in v1.1.115:
https://github.com/ShiftLeftSecurity/joern/pull/471

**Comment by astrogeco** [comment]
It works! Had some other warnings in the test program but I'll post those as a separate issue
